obspy/io/xseed/tests/test_parser.py

```diff
@@ -533,6 +533,110 @@ def test_create_resp_from_xseed(self):
             # create RESP files
             sp2.get_resp()
 
+    def test_read_resp(self):
+        """
+        Tests reading a respfile by calling Parser(filename)
+        """
+        sts2_resp_file = os.path.join(self.path,
+                                      'RESP.XX.NS085..BHZ.STS2_gen3.120.1500')
+        rt130_resp_file = os.path.join(self.path,
+                                       'RESP.XX.NR008..HHZ.130.1.100')
+        for filename in (sts2_resp_file, rt130_resp_file):
+            p = Parser(filename)
+            p.get_resp()
+
+    def test_read_resp_data(self):
+        """
+        Tests reading a resp string by calling Parser(string)
+        """
+        sts2_resp_file = os.path.join(self.path,
+                                      'RESP.XX.NS085..BHZ.STS2_gen3.120.1500')
+        rt130_resp_file = os.path.join(self.path,
+                                       'RESP.XX.NR008..HHZ.130.1.100')
+        for filename in (sts2_resp_file, rt130_resp_file):
+            with open(filename, 'rt') as f:
+                data = f.read().encode('utf-8')
+            p = Parser(data)
+            p.get_resp()
+
+    def test_resp_round_trip(self):
+        single_seed = os.path.join(
+            self.path,
+            '../../../../',
+            'core/tests/data/IRIS_single_channel_with_response.seed')
+        # Make parser and get resp from SEED
+        seed_p = Parser(single_seed)
+        resp_from_seed = seed_p.get_resp()[0][1]
+        resp_from_seed.seek(0)
+        resp_from_seed = resp_from_seed.read()
+
+        # make parser from resp made above and make a resp from it
+        resp_p = Parser(resp_from_seed)
+        resp_from_resp = resp_p.get_resp()[0][1]
+        resp_from_resp.seek(0)
+        resp_from_resp = resp_from_resp.read()
+
+        # Strip out string constants that differ.
+        ignored_fields = ['B054F05', 'B054F06', 'B053F05', 'B053F06']
```

Comment:
Looks like counts and m/s can be addressed with case ignore
volts  gets `emf in `
```
17,18c17,18
< B053F05     Response in units lookup:              M/S - velocity in meters per second
< B053F06     Response out units lookup:             V - emf in volts
---
> B053F05     Response in units lookup:              M/S - Velocity in Meters Per Second
> B053F06     Response out units lookup:             V - Volts
50,51c50,51
< B054F05     Response in units lookup:              V - emf in volts
< B054F06     Response out units lookup:             COUNTS - digital counts
---
> B054F05     Response in units lookup:              V - Volts
> B054F06     Response out units lookup:             COUNTS - Digital Counts
81,82c81,82
< B054F05     Response in units lookup:              COUNTS - digital counts
< B054F06     Response out units lookup:             COUNTS - digital counts
---
> B054F05     Response in units lookup:              COUNTS - Digital Counts
> B054F06     Response out units lookup:             COUNTS - Digital Counts
```